backend/lib/models/User.js

```diff
@@ -1,33 +1,32 @@
-const { model, Schema } = require("mongoose");
+const { Schema, model } = require("mongoose");
+const { schema: locationSchema } = require("./Location");
+const { isValidEmail } = require("../utils");
 
-const UserSchema = new Schema(
+const userSchema = new Schema(
   {
-    _id: {
+    about: { maxLength: 100, trim: true, type: String },
+    email: {
```

Comment:
yea so I'm wondering about emails, if we should save them in our own database
and if we should allow updating through the API
We currently get email also from Auth0 users, also with social media accounts, so I would prefer to use those, and we could save them from the Auth0 user instead of through the API, or at least has to be checked that it's the same anyway.

Reply:
About not storing the email on our side, I think storing it is not a big deal and will be really useful for BI/Analytics reports where we need to have user contact info. Those will be routine at  mkt actions and removing email from ou database would make us depend on Auth0 for that, so I think we should keep them at our side too.

I think restricting the update through the API and linking it to the social email is okay as long as we keep it at our side synced to use them at metabase and other BI reports.

Reply:
yep if email for org is not really used at the moment it's not a problem, we can check it later